docs/src/getting-started/basics.md

```diff
@@ -276,6 +276,35 @@ let contract_id = "0x0123..." // Your contract ID as a string.
 let connected_contract_instance = MyContract::new(contract_id, wallet);
 ```
 
+## Getting the contract call outputs
+
+### Contract call success
+
+If the contract call succeeded, getting the receipts and logs is done this way:
+
+```rust,ignore
+let result = contract_instance.my_method(args).call().await?;
+logs = result.logs.unwrap() // This gives out the decoded hex LOGD logs
```

Comment:
Wait, why is this a `Result`? If the transaction completed execution then it will have logs, no matter what. If the transaction was invalid, then it won't even run, and won't have receipts other. The entire `result` should be a `Result`, not just the `logs`.

Reply:
`result` here is a `CallResponse`, the `?` is there to unwrap it from a `Result<CallResponse, ContractCallError>`. The `logs.unwrap()` is indeed misguided, because it's an `unwrap` on an `Options<Vec<Receipt>>`.
I'm not sure what the example should be then, the point was to show how to "get the receipts" in both cases.

Reply:
I can change `result` into `response` to make it more clear, you're right.

Reply:
Shouldn't it be `result.unwrap().logs` then?

Reply:
An empty `Vec` makes more sense than an `Option`.

Reply:
Agreed, hadn't thought about it

Reply:
#343 to rename